lonestar/analytics/distributed/bc/README.md

```diff
@@ -19,29 +21,26 @@ BUILD
 
 1. Run cmake at BUILD directory (refer to top-level README for cmake instructions).
 
-2. Run `cd <BUILD>/dist-apps/; make -j bc_level
+2. Run `cd <BUILD>/lonestar/analytics/distributed/betweennesscentrality; make -j
 
 RUN
 --------------------------------------------------------------------------------
 
 To run solving all sources, use the following:
-`./bc_level <input-graph> -t=<num-threads>`
+`./betweennesscentrality-level-dist <input-graph> -t=<num-threads>`
 
 To run for the first n sources, use the following:
-`./bc_level <input-graph> -t=<num-threads> -numOfSources=n`
+`./betweennesscentrality-level-dist <input-graph> -t=<num-threads> -numOfSources=n`
 
 To run on 3 hosts h1, h2, and h3 with a Cartesian vertex cut partition, use the 
 following:
-`mpirun -n=3 -hosts=h1,h2,h3 ./bc_level <input-graph> -t=<num-threads> -partition=cvc`
+`mpirun -n=3 -hosts=h1,h2,h3 ./betweennesscentrality-level-dist <input-graph> -t=<num-threads> -partition=cvc`
 
 PERFORMANCE
 --------------------------------------------------------------------------------
 
-This implementation does not perform well on high diameter graphs such as road
```

Comment:
Readd this line, but change it to "level" does not perform well on high diameter graphs.

Reply:
I'll handle this later.